### PR TITLE
release 2_4_18

### PR DESCRIPTION
**Pull request by 48d90782**
update rr to 1.8.1
update broadcast to version w\o hash

### CODE REVIEW

File: bin/spiral [hunk 1]
```diff
@@ -9,8 +9,8 @@ declare(strict_types=1);
 
 foreach ([
              __DIR__ . '/../../../autoload.php',
-             __DIR__ . '/../vendor/autoload.php',
-             __DIR__ . '/vendor/autoload.php'
+             __DIR__ . '/../vendor_php/autoload.php',
```
Comment (wolfy-j) [inline bin/spiral:12]: This is not correct. The vendor path will only change for local development, the actual project will still use vendor.

File: src/Bootloader/Security/ValidationBootloader.php [hunk 2]
```diff
@@ -114,7 +114,7 @@ public function boot(TokenizerBootloader $tokenizer): void
             ]
         );
 
-        $tokenizer->addDirectory(directory('vendor') . 'spiral/validation/src/');
```
Comment (wolfy-j) [inline src/Bootloader/Security/ValidationBootloader.php:117]: Same

File: src/Framework/Kernel.php [hunk 3]
```diff
@@ -57,7 +57,7 @@ protected function mapDirectories(array $directories): array
                 'public'    => $directories['root'] . '/public/',
 
                 // vendor libraries
-                'vendor'    => $directories['root'] . '/vendor/',
```
Comment (wolfy-j) [inline src/Framework/Kernel.php:60]: Same

File: tests/bootstrap.php [hunk 4]
```diff
@@ -12,4 +12,4 @@
 ini_set('display_errors', '1');
 
 //Composer
-require dirname(__DIR__) . '/vendor/autoload.php';
```
Comment (wolfy-j) [inline tests/bootstrap.php:15]: This one can stay with vendor_php

File: bin/spiral (hunk 1 above)
Comment (48d90782) [reply bin/spiral]: done

File: src/Framework/Kernel.php (hunk 3 above)
Comment (48d90782) [reply src/Framework/Kernel.php]: done

File: tests/bootstrap.php (hunk 4 above)
Comment (48d90782) [reply tests/bootstrap.php]: done

File: src/Bootloader/Security/ValidationBootloader.php (hunk 2 above)
Comment (48d90782) [reply src/Bootloader/Security/ValidationBootloader.php]: done

**Pull request merged**